pkg/util/slice/slice.go

```diff
@@ -68,3 +68,16 @@ func ContainsString(slice []string, s string, modifier func(s string) string) bo
 	}
 	return false
 }
+
+// Deduplicate remove the duplicate value in the slice and keep the order unchanged
```

Comment:
Deduplicate removes any duplicated values and returns a new slice, keeping the order unchanged.